tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +66,70 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// GetVDVSPlugin - get vDVS plugin id
+func GetVDVSPlugin(ip string) (string, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if misc.FormatOutput(out) == "" {
+		return "", fmt.Errorf("vDVS plugin unavailable")
+	}
+	return strings.Fields(misc.FormatOutput(out))[0], err
+}
+
+// StartVDVSPlugin - manually start vDVS plugin
+func StartVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Manually starting vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.StartVDVSPlugin)
+	return out, err
+}
+
+// KillVDVSPlugin - kill vDVS plugin. It is restarted automatically
+func KillVDVSPlugin(ip, pluginID string) ([]byte, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+	time.Sleep(2 * time.Second)
```

Comment:
it will be lot easier to adjust wait time in future from central place. 

You may want to consider moving this into separate util.